hpi-archetype/pom.xml

```diff
@@ -15,15 +14,33 @@
   <!-- $ --><version>1.0-SNAPSHOT</version><!-- /$ -->
   <packaging>hpi</packaging>
 
+  <properties>
+    <!-- Baseline Jenkins version you use to build and test the plugin. Users must have this version or newer to run. -->
+    <jenkins.version>1.580.1</jenkins.version>
+    <!-- Java Level to use. Default is 7, but was only required by Jenkins >= 1.614. -->
+    <java.level>6</java.level>
+    <!-- Other properties you may want to use:
+         ~ jenkins-test-harness.version: The JTH version used to test plugin. The default is ${jenkins.version}.
+         ~ hpi-plugin.version: The HPI Maven Plugin version used by the plugin. Current default is 1.115.
+         ~ stapler-plugin.version: The Stapler Maven plugin version required by the plugin. Current default is 1.17
+    -->
+  </properties>
+
   <name>TODO Plugin</name>
   <description>TODO</description>
   <url>https://wiki.jenkins-ci.org/display/JENKINS/TODO+Plugin</url>
+
+  <!-- The default licence for Jenkins OSS Plugins is MIT. Uncomment or substitute for the applicable one.
```

Comment:
Was intentional for this to be uncommented in the _archetype_. I do not think it should be silently _inherited_ from the parent.